services/monitor/monitor.sh

```
@@ -0,0 +1,368 @@
+        2>/dev/null || echo ""
+}
+
+# Executes docker compose with proper path resolution for bind mounts
+# Uses host project directory for path resolution while accessing compose files from container
+execute_compose_command() {
+    cd /workspace || {
+        error "Failed to change to workspace directory"
+        return 1
+    }
+
+    local host_project_dir=$(get_host_project_directory)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)